launch/auton_sim.launch

```diff
@@ -17,16 +17,23 @@
     </include>
 
     <!-- visual odometry node to provide localization in odom frame -->
-    <node if="$(arg use_odom)" name="stereo_odometry" pkg="rtabmap_ros" type="stereo_odometry"
-          output="screen" launch-prefix="bash -c 'sleep 10; $0 $@'">
-        <remap from="odom" to="/odometry"/>
-        <param name="frame_id" type="string" value="base_link"/>
-        <param name="odom_frame_id" type="string" value="odom"/>
```

Comment:
why are you getting rid of these params?

Reply:
Ig the odom remap is not, I can add that back